Makefile

```
@@ -0,0 +1,23 @@
+CHART_NAME_1 ?= cray-vpa
+CHART_PATH ?= kubernetes
+CHART_VERSION_1 ?= local
+CHART_VERSION_2 ?= local
+
+HELM_UNITTEST_IMAGE ?= quintush/helm-unittest:3.3.0-0.2.5
+
+
+all : charts
+charts: chart_setup chart_packages chart_tests
+chart_packages: chart_package_1 chart_package_2
```

Dont need the "_1" theres only one chart or the second chart
```suggestion
chart_packages: chart_package
```